test/hotspot/jtreg/compiler/c2/irTests/ModINodeIdealizationTests.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2022, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```
Check the others too. I don't think a new file requires old dates ;)

Reply:
The basic structure is based of the existing tests (I think mostly DivINodeIdealizationTests) so I was thinking this is a derivative work.